mission-control/docs/reference/connections/github.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Github
+---
+
+Github connection can be used for playbook's GitOps actions and Health Checks
+
```

Comment:
Can we add details on how to get the token? and use links for actions and applicable health checks

Reply:
Added 👍🏽